src/ucp/tag/rndv.c

```diff
@@ -1324,29 +1641,30 @@ UCS_PROFILE_FUNC(ucs_status_t, ucp_rndv_rtr_handler,
                       ucp_ep_peer_name(ep), ucs_status_string(status));
         }
 
-        is_pipeline_rndv = ((!UCP_MEM_IS_ACCESSIBLE_FROM_CPU(sreq->send.mem_type) ||
-                             (sreq->send.length != rndv_rtr_hdr->size)) &&
-                            (context->config.ext.rndv_mode != UCP_RNDV_MODE_PUT_ZCOPY));
+        is_put_pipeline_rndv = ((!UCP_MEM_IS_ACCESSIBLE_FROM_CPU(sreq->send.mem_type) ||
```

Comment:
no need for `()` around `!UCP_MEM_IS_ACCESSIBLE_FROM_CPU(sreq->send.mem_type)`

Reply:
I do not have `()` around this but have around `(cond1 || cond2)` . am I missing something?

Reply:
right, my fault